src/battery-check/battery-check.c

```diff
@@ -153,9 +136,9 @@ static int run(int argc, char *argv[]) {
 
         log_setup();
 
-        r = proc_cmdline_parse(parse_proc_cmdline_item, NULL, PROC_CMDLINE_STRIP_RD_PREFIX);
+        r = proc_cmdline_get_bool("systemd.battery-check", PROC_CMDLINE_STRIP_RD_PREFIX, &arg_doit);
```

Comment:
Be careful, this disables battery check when the option is not specified. Please re-enable when `r==0`.